lib/objectid.js

```diff
@@ -315,6 +315,7 @@ class ObjectId {
    * Checks if a value is a valid bson ObjectId
    *
    * @method
+   * @param {object} id ObjectId instance to validate.
```

Comment:
Since it is valid to pass any type in to the function, could you change the type field from `object` to `*`?

Reply:
Sure thing, thanks!